internal/service/streaminstance/data_source_stream_instance_test.go

```diff
@@ -24,7 +24,10 @@ func TestAccStreamDSStreamInstance_basic(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				Config: streamInstanceDataSourceConfig(orgID, projectName, instanceName, region, cloudProvider),
-				Check:  streamInstanceAttributeChecks(dataSourceName, orgID, projectName, instanceName, region, cloudProvider),
+				Check: resource.ComposeTestCheckFunc(
+					streamInstanceAttributeChecks(dataSourceName, instanceName, region, cloudProvider),
+					resource.TestCheckResourceAttr(dataSourceName, "stream_config.tier", "SP30"),
```

Comment:
[q] why can't this check be included in `streamInstanceAttributeChecks`?

Reply:
Didn't include it there because migration tests does not need that check, because it uses a previous version of the provider which does not have that attribute. Hope it makes sense

Reply:
if this is a new attribute, consider using SkipIfVersionBelow and/or IsProviderVersionAtLeast in their mig tests, you can look for examples of use in the code